src/etc/inc/openvpn.tls-verify.php

```diff
@@ -34,6 +34,8 @@
 require_once("config.inc");
 require_once("interfaces.inc");
 
+global $config;
```

Comment:
global declaration is not needed outside a function

Reply:
Hm, right
fixed